### PR TITLE
Document BUILDAH_* environment variables in buildah bud --help output

### PR DESCRIPTION
**Pull request by rhatdan**
We also want to show the default settings.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: pkg/cli/common.go [hunk 1]
```diff
@@ -260,3 +261,21 @@ func UseLayers() bool {
...
+	return "oci"
+}
+
+// DefaultIsolation returns the default image format
+func DefaultIsolation() string {
+	isolation := os.Getenv("BUILDAH_ISOLATION")
+	if isolation != "" {
+		return isolation
+	}
+	return ""
```
Comment (vrothberg) [inline pkg/cli/common.go]: According to `buildah-bud(1)` the default should be `"oci"`.

Comment (rhatdan) [reply pkg/cli/common.go]: It would still be, but I guess I can make it more clear.

**Comment by vrothberg** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**